frontend/src/views/tasks/TaskDetailView.vue

```diff
@@ -946,6 +953,18 @@ function setRelatedTasksActive() {
 		}
 	}
 }
+
+async function duplicateTask() {
+	const clone = await taskStore.duplicateTask(task.value)
+	if (clone != null) {
```

Comment:
What could be a case where the clone returned is `null`? Can we handle that explicitly?

And please use `!==`:
```suggestion
	if (clone !== null) {
```

Reply:
If the backed will return error, then clone will be null. There are error message displayed, so I do not think that it requires additional handling.